portfolio/src/main/java/com/google/sps/data/CommentPersistHelper.java

```diff
@@ -0,0 +1,106 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Collections;
+import javax.servlet.http.HttpServletRequest;
+import com.google.gson.Gson;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.appengine.api.datastore.FetchOptions;
+import com.google.appengine.api.datastore.Key;
+
+/** Keeps track of persisted comments with ability to add/remove comments */
+public class CommentPersistHelper {
+  private final DatastoreService datastore;
+  private final List<Comment> comments;
+
+  public CommentPersistHelper() {
+    datastore = DatastoreServiceFactory.getDatastoreService();
+    comments = new ArrayList<>();
+    loadComments();
+  }
```

Comment:
since loadComments is publicly available, I would suggest instead of loading comments in the constructor (side effects). 

this enables you to have a commentPersistHelper that you can create without causing a strain on your DB.